apps/src/templates/sectionAssessments/assessmentDataShapes.js

```diff
@@ -18,3 +12,29 @@ export const questionDataPropType = PropTypes.shape({
   answers: PropTypes.arrayOf(answerDataPropType),
   notAnswered: PropTypes.number.isRequired,
 });
+
+// Shapes for single student multiple choice tables
+
+// Represents an individual students response to a single
+// multiple choice question.
+export const studentResponsePropType = PropTypes.shape({
+  isCorrect: PropTypes.bool,
+  answers: PropTypes.string,
+});
+
+// Represents a single student and a set of there answers for
+// a single assessment's multiple choice questions
```

Comment:
Spelling: `there/their`.

Would it be accurate to say "Represents one student and _all_ of their multiple-choice answers for one assessment"?

With all of these prop types, let's establish some shared terminology.  Having a little glossary would help, something like this:

- **answer** - a single selected choice in a multiple-choice question.
- **response** - a set of selected choices in a multiple-choice question.
- ...